docker/centos/7/xpu_ispc_build/Dockerfile

```diff
@@ -46,6 +46,11 @@ ENV XE_DEPS=/usr/local/deps
 # http://www.intel.com/software/sde and download the latest version,
 # extract it, add to path and set SDE_HOME.
 
+#
```

Comment:
Supposed to be in the beginning of the file

Reply:
I have no idea how it appeared here :)